Bottom: 7255a314df659869632332ca977eff288198b6f0
Top:    cf1f2352523b86bab1562a94164181f0a9e7b5a0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 13:23:31 +0100

Refresh of expose-gathering-variable-yuck

---

diff --git a/src/draw.c b/src/draw.c
index 360df90..0a8b81e 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -71,7 +71,7 @@ RCSID ("$Id$");
  * some local identifiers
  */
 static BoxType Block = {MAXINT, MAXINT, -MAXINT, -MAXINT};
-static bool Gathering = true;
+bool Gathering = true;
 
 static int doing_pinout = 0;
 static bool doing_assy = false;
